### ISSUE TITLE
range with step

### ISSUE THREAD

**Issue by rncar** [janet-lang/janet#50]
```
> (range 0 10 3)
@[0                                                                        
  nil                                                                                                       
  nil                                                                                        
  3                                                                         
  nil                                                                       
  nil                                                                                   
  6                                                                                                          
  nil                                                                                   
  nil                                                                                                        
  9]
```
is this correct ?

**Comment by bakpakin** [comment]
Haha no.

**Comment by bakpakin** [comment]
Fixed in a982f351d71aa655491a1089046b0a40d809755d.
